### PR TITLE
kthfromend method complete.  tests pass.

### PR DESCRIPTION
**Pull request by Bravelemming**

### CODE REVIEW

File: code401challenges/src/test/java/linkedList/LinkedListTest.java [hunk 1]
```diff
@@ -95,4 +95,17 @@ public void insertAfterTest() {
 
         assertEquals(expected, actual);
     }
+
```
Comment (Tw1stedRain) [inline code401challenges/src/test/java/linkedList/LinkedListTest.java]: this is only one test for the kth from the end method. i would like to see more tests, get some edge cases in there, to the same thing a couple times just slightly different. anything that shows that it works all the time, not just this one time
